src/bin.ts

```diff
@@ -236,6 +238,50 @@ const deploy = async ({ yes, bucket }: { yes: boolean, bucket: string }) => {
             }));
         });
 
+        uploadQueue.push(...permanentRedirects.map(redirect =>
+            asyncify(async () => {
+                const { fromPath, toPath: redirectLocation } = redirect
```

Comment:
There are a few lines (such as this one) which are missing semicolons.

(We're planning on adding ESLint but haven't gotten around to it)